common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1105,6 +1108,107 @@ def test_discussion_privileged_user_can_edit_team(self, role):
         )
 
 
+@ddt.ddt
+class EditMembershipTest(TeamFormActions):
+    """
+    Tests for administrating from the team membership page
+    """
+
+    def setUp(self):
+        super(EditMembershipTest, self).setUp()
+
+        self.set_team_configuration(
+            {'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]},
+            global_staff=True
+        )
+        #TODO: update this to whatever it ended up being renamed in Peter's code
+        self.create_edit_team_page = CreateOrEditTeamPage(self.browser, self.course_id, self.topic)
+        self.team = self.create_teams(self.topic, num_teams=1)[0]
+
+        self.edit_membership_page = EditMembershipPage(self.browser, self.course_id, self.team)
+        self.team_page = TeamPage(self.browser, self.course_id, team=self.team)
+        self.team_page.visit()
+
+    @ddt.data('Moderator', 'Community TA', 'Administrator')
```

Comment:
Add None like Peter did in his tests so you can also test the is_staff version (not discussion admin/moderator).